core/art/views.py

```diff
@@ -1265,6 +1316,8 @@ def registerARTTest(request):
         _logger.exception('Failed to parse date from nightly_tag')
 
     # Check whether the pandaid has been registered already
+    if test_type and test_type == 'local':
+        computingsite = "ART Local"
```

Comment:
this can be also moved up to ~1187, to shorten code and avoid this if statement.

Reply:
Move as suggested.